app/components/article-featured-video.js

```diff
@@ -154,12 +186,48 @@ export default Component.extend(
 			});
 		},
 
+		canVideoDrawerShow() {
+			return !this.get('isVideoDrawerVisible') &&
+				!this.get('videoDrawerDismissed') &&
+				!this.get('isPlayerLoading') &&
+				this.player.getState() !== window.OO.STATE.PLAYING;
+		},
+
+		closeVideoDrawer() {
+			if (this.get('isVideoDrawerVisible')) {
+				this.set('isVideoDrawerVisible', false);
+				this.toggleSiteHeadShadow(true);
+			}
+		},
+
 		actions: {
 			playVideo() {
 				if (this.player) {
+					if (this.get('isVideoDrawerVisible')) {
+						this.player.mb.publish(window.OO.EVENTS.WILL_CHANGE_FULLSCREEN, true);
+						track({
+							action: trackActions.click,
+							category: 'article-video',
+							label: 'on-scroll-bar'
+						});
+					}
+
 					this.set('isPlayed', true);
+					this.closeVideoDrawer();
 					this.player.play();
 				}
+			},
+			closeVideoDrawer() {
```

Comment:
We have `this.closeVideoDrawer` and `this.actions.closeVideoDrawer`. I'm pretty sure someone at some point will mix them up. Can we distinct them somehow?